doc/internal/ReleaseInstructions.md

```diff
@@ -275,6 +276,7 @@ git checkout master
 git pull --ff-only upstream master
 
 # Example:
```

Comment:
`JAVA_VERSION=2.1.1-SNAPSHOT` on the next line is meant as the example.

You can also see that in the rendered version: http://vitess.io/internal/release-instructions.html#bump-java-snapshot-version

Do you think that's not clear enough?